src/runtime_src/doc/toc/xrt_native_apis.rst

```diff
@@ -254,7 +254,18 @@ XRT provides ``xrt::bo::copy()`` API for deep copy between the two buffer object
            
            dst_buffer.copy(src_buffer, copy_size_in_bytes);
 
-The API ``xrt::bo::copy()`` also has overloaded versions to provide a different offset than 0 for both the source and the destination buffer. 
+The API ``xrt::bo::copy()`` also has overloaded versions to provide a different offset than 0 for both the source and the destination buffer.
+
+IV. Data trasfer from/to device only buffer using read/write API's
+******************************************************************
+
+To avoid of host memcpy with the usage of ``xrt::bo::sync()``, Vitis AI runtime (VART) currently leverages ``xclUnmgdPwrite()`` and ``xclUnmgdPread()`` to perform PCIe DMA data transfer between host and device so as to achieve better performance.
```

Comment:
We dont need to mention about xclUnmgtPread and xclUnmgdPwrite in our documentation.
Remove the above sentence completely.

Reply:
We dont need to mention about VART also.

Reply:
After looking at overall documentation. I feel, it is better to add this info in Section "I. Data transfer between host and device by Buffer read/write API" at the end.